compiler/x/codegen/X86Debug.cpp

```diff
@@ -2694,886 +2694,18 @@ TR_Debug::print(TR::FILE *pOutFile, TR::AMD64WriteBarrierSnippet * snippet)
 
 #endif
 
-static const char * opCodeToNameMap[IA32NumOpCodes] =
+static const char * opCodeToNameMap[] =
    {
```

Comment:
For a future follow-on item, it should be possible to roll these names into your unified table as well.